modules/apps/layout/layout-content-page-editor-web-test/src/testIntegration/java/com/liferay/layout/content/page/editor/web/internal/portlet/action/test/AddSegmentsExperienceMVCActionCommandTest.java

```diff
@@ -95,8 +103,24 @@ public void testAddSegmentsExperiment() throws Exception {
 		SegmentsEntry segmentsEntry = SegmentsTestUtil.addSegmentsEntry(
 			_group.getGroupId());
 
-		JSONObject responseJSONObject = _addSegmentsExperience(
-			name, segmentsEntry.getSegmentsEntryId());
+		SegmentsExperienceTestUtil.addSegmentsExperienceData(
+			"fragment_composition_with_a_card.json", _draftLayout,
```

Comment:
It is hard for the reviewer that yourself for the future to read the method SegmentsExperienceTestUtil.addSegmentsExperienceData that receives the parameter "fragment_composition_with_a_card.json", that is the name of a file that should be located in a very specific path, better to use InputStream or File.

Reply:
Changed ❤️